ebs_cleanup/app.py

```diff
@@ -0,0 +1,85 @@
+import datetime
+import json
+import logging
+
+import boto3
+
+
+LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.INFO)
+
+
+def local_volume_filter(volumes):
+    output = []
+
+    for v in volumes:
+        # double-check for attachments
+        if v.attachments:
+            LOG.error(f"Volume {v.volume_id} has attachments: {v.attachments}")
```

Comment:
it's logged as an error because we don't expect to see any attachments listed if the status is `available`, though I suppose that adding `error` as an acceptable state means that some of those volumes might have attachments